### PR TITLE
[docs] [2.23] Add pg_cron extension docs

### PR DESCRIPTION
**Pull request by hari90**

### CODE REVIEW

**Review by aishwarya24 — APPROVED** [review]
LGTM

File: docs/content/preview/explore/ysql-language-features/pg-extensions/extension-pgcron.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+./bin/yugabyted start --master_flags "enable_pg_cron=true,allowed_preview_flags_csv=enable_pg_cron" --tserver_flags "enable_pg_cron=true,allowed_preview_flags_csv=enable_pg_cron" --ui false
+```
+
+### 2. Set the pg_cron database (optional)
+By default, pg_cron runs on the `yugabyte` database. You can change this by setting the `ysql_cron_database_name` flag to your desired database name.
+{{< note >}}
+- The database can be created after setting the flag.
+- In order to change the database after the extension is created, you must drop the extension before changing the flag.
+{{< /note >}}
+
```
Comment (ddhodge) [inline docs/content/preview/explore/ysql-language-features/pg-extensions/extension-pgcron.md:38]: Example? Do you set this at start also? In which case I'd add this to previous section
You can only run 1 cron, and only on 1 database?

File: docs/content/preview/explore/ysql-language-features/pg-extensions/extension-pgcron.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+### 3. Create and use the pg_cron extension
+Create the extension as superuser on the cron database.
```
Comment (ddhodge) [inline docs/content/preview/explore/ysql-language-features/pg-extensions/extension-pgcron.md:41]: only superuser? shouldn't it also be available to yb_extension

File: docs/content/preview/explore/ysql-language-features/pg-extensions/extension-pgcron.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+  preview:
+    identifier: extension-pgcron
+    parent: pg-extensions
+    weight: 20
+type: docs
+---
+
+The [pg_cron](https://github.com/citusdata/pg_cron) extension provides a cron-based job scheduler that runs inside the database. It uses the same syntax as regular cron, and allows you to schedule YSQL commands directly from the database. You can also use '[1-59] seconds' to schedule a job based on an interval.
+
+YugabyteDB supports all features of the pg_cron extension. Although YugabyteDB is a distributed database that operates on multiple nodes, pg_cron only runs on one of these nodes, called the pg_cron leader. Only the pg_cron leader schedules and runs the cron jobs. The queries executed by jobs do take advantage of all available resources in the cluster.
```
Comment (hari90) [inline docs/content/preview/explore/ysql-language-features/pg-extensions/extension-pgcron.md:17]: > YugabyteDB supports all features of the pg_cron extension.

@ddhodge 
This line repeats in `Use pg_cron`

**Review by ddhodge — APPROVED** [review]

**Pull request merged**